### PR TITLE
Fixup inconsistent aliases

### LINKED ISSUE #329: Investigate inconsistent aliases
I used @oxalorg's inconsistent aliases script to detect inconsistent aliases in Kaocha.

```
Running analysis on lambdaisland/kaocha/src
clojure.spec.alpha                                 -> s, spec
clojure.test                                       -> t, test
```

```
Running analysis on lambdaisland/kaocha/test
kaocha.report                                      -> report, r
kaocha.output                                      -> output, out
kaocha.hierarchy                                   -> hierarchy, h
kaocha.config                                      -> config, c
```

### PR DESCRIPTION
**Pull request by solar05**
Fixes #329

### CODE REVIEW

File: test/unit/kaocha/watch_test.clj [hunk 1]
```diff
@@ -1,7 +1,6 @@
 (ns kaocha.watch-test
   (:require [clojure.test :refer :all]
             [clojure.spec.alpha :as s]
-            [kaocha.config :as c]
             [kaocha.watch :as w]
```
Comment (solar05) [inline test/unit/kaocha/watch_test.clj:4]: Removed, because the same ns is imported below

**Comment by alysbrooks** [comment]
Curious why you picked the single-letter alias for some and not others. I'm primarily wondering about `s` for `spec`. I would probably pick `spec` for that require, but if the majority of namespaces already use `s` (or you have another compelling reason), then I'm fine with it.

**Comment by alysbrooks** [comment]
FYI, the relevant part of our Clojure Style Guide:

> Some namespaces that are very core to an application and used all over the place are better off having a shorter alias used consistently.

We should probably make this easier to find for contributors. :thinking:

**Comment by alysbrooks** [comment]
The changes look good overall. Thank you for the PR! (Sorry for not mentioning that in my initial comment.)

**Comment by alysbrooks** [comment]
Released in [v1.78.1249](https://github.com/lambdaisland/kaocha/releases/tag/v1.78.1249)

```clj
[lambdaisland/kaocha "1.78.1249"]                 ;; deps.edn
{lambdaisland/kaocha {:mvn/version "1.78.1249"}}  ;; project.clj
```

**Pull request merged**